metagraph/src/cli/clean.cpp

```diff
@@ -135,16 +135,17 @@ int clean_graph(Config *config) {
                                           *node_weights,
                                           config->min_unitig_median_kmer_abundance))
                     callback(unitig, path);
-            }, num_threads, config->min_tip_size, graph->is_canonical_mode());
+            }, num_threads, config->min_tip_size, graph->get_mode() == graph::DeBruijnGraph::CANONICAL);
```

Comment:
similar idea here, why not just pass the mode and let `call_unitigs` figure out how to interpret it?

Reply:
I don't see why that's better. We need 2 regimes here, the current boolean flag does the job perfectly.

Reply:
Besides, the graph already knows its state.
Do you propose to remove this flag and the choice which sequences to extract?